Kconfig

```diff
@@ -38,14 +38,6 @@ config HOST_PTABLE
 	bool
 	default n
```

Comment:
This is a major change and I would really like the commit message to explain the new implementation better. And also explain why the new implementation is better than what we have now.

I have a deja-vu that I already commented about this long time ago :)

Reply:
I've rephrased the commit description to include things you want. I hope now it's enough.